x/staging/posts/client/cli/query.go

```diff
@@ -253,11 +254,47 @@ func GetCmdQueryParams() *cobra.Command {
 	return cmd
 }
 
+// GetCmdQueryPostReactions returns the command allowing to query the reactions of a post
+func GetCmdQueryPostReactions() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "post-reactions [post-id]",
```

Comment:
I think we can rename this to simply be `reactions` since the registered reactions is already `registered-reactions`. We don't need to specify these are posts reactions IMO